wp-content/themes/csisnuclearnetwork/inc/template-tags.php

```diff
@@ -159,12 +159,19 @@ function nuclearnetwork_posted_on( $date_format = null ) {
 	}
 
 	$date = get_option( 'date_format' );
+	$post_type = get_post_type();
 
 	if  ( $date_format ) {
 		$date = $date_format;
 	}
 
-	echo '<div class="post-meta post-meta__date">' . get_the_time( $date ) . '</div>';
+	if ( $post_type === 'news' ) {
+		echo '<div class="post-meta post-meta__date">' . get_the_time( $date ) . '</div>';
+	} elseif ( $post_type === 'events' || $post_type === 'programs' || $post_type === 'projects' ) {
```

Comment:
```suggestion
	} elseif ( in_array( $post_type, array( 'events', 'programs', 'projects') )  ) {
```

If you find yourself checking against multiple post types, it's better to check if they're in an array to simplify your if statements. See more info about [`in_array` here](https://www.php.net/manual/en/function.in-array.php).